prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -128,157 +162,144 @@ def smart_charging(
             # trip amount for each vehicle
             total_trips = int(newdata.iloc[i, newdata.columns.get_loc("total_trips")])
 
-            if data_day[i] == input_day[day_iter]:
-
-                # only home based trips
-                if (
-                    newdata.iloc[i, newdata.columns.get_loc("why_from")]
-                    * newdata.iloc[
-                        i + total_trips - 1, newdata.columns.get_loc("dwell_location")
-                    ]
-                    == 1
-                ):
```

Comment:
We no longer need this logic or it has been moved somewhere else?

Reply:
Lines `only home based trips` block is a filtering process of the input data that we'll be moving to an early point in the code workflow.  It's also one of the sources of the smart charging scaling factor bug.

Reply:
Good to know!